db/migrations/00010.add_fkc_course_usage_table.py

```diff
@@ -0,0 +1,14 @@
+#
+# file: migrations/00010.add_fkc_course_usage_table.py
```

Comment:
Can you fix the numbering here and in the file name so it's `0010`?

Reply:
Done